server/src/HatoholArmPluginGateHAPI2.cc

```diff
@@ -631,8 +636,13 @@ struct HatoholArmPluginGateHAPI2::Impl
 		virtual void onTimeout(void) override
```

Comment:
`FetchProcedureCallback` does not have setArmInfoStatus or similiar method like `ExchangeProfileCallback`'s `setArmInfoStatus`.
It is strange for me.
Because once `ExchangeProfileCallback` executed, it does not recover normal state except for calling  `ExchangeProfileCallback`'s `setArmInfoStatus`.